src/main/webapp/search-interview.html

```diff
@@ -0,0 +1,107 @@
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+      integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+      integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
+    <script src="search-interview.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <!-- TODO: add navbar here. -->
+      <h1 class="text-center">Search for an Interview</h1>
+      <h5 class="text-center lead">Search for possible times in which you will be interviewed!</h5>
+      <br>
+      <form>
+        <div class="form-group">
+          <!-- TODO: add parameters for this form. -->
+          <button type="button" class="btn btn-primary" onclick="loadInterviews()">Search</button>
+        </div>
+      </form>
+      <br>      
+      <section hidden id="search-results">
+        <!--TODO: Include Timezone consideration; Add pagination; Render these interview times dynamically;-->
+        <form>
+          <div class="form-row">
+            <div class="col-3 date-label">
+              <label>Monday 6/29</label>
+            </div>
+            <div class="col-5">
+              <select class="form-control">
+                <option>8:15 AM - 9:15 PM</option>
+                <option>10:30 AM - 11:30 AM</option>
+                <option>1:00 PM - 2:00 PM</option>
+              </select>
+            </div>
+            <div class="col-4">
+              <button type="button" class="btn btn-primary mb-2">Select</button>
+            </div>
+          </div>
+        </form>
+        <br>
+        <form>
+          <div class="form-row">
+            <div class="col-3 date-label">
+              <label>Wednesday 6/30</label>
+            </div>
+            <div class="col-5">
+              <select class="form-control">
+                <option>6:30 PM - 7:30 PM</option>
+              </select>
+            </div>
+            <div class="col-4">
+              <button type="button" class="btn btn-primary mb-2">Select</button>
+            </div>
+          </div>
+        </form>
+        <br>     
+        <form>
+          <div class="form-row">
+            <div class="col-3 date-label">
+              <label>Thursday 7/1</label>
+            </div>
+            <div class="col-5">
+              <select class="form-control">
+                <option>9:30 AM - 10:30 AM</option>
+                <option>6:30 PM - 7:30 PM</option>
+              </select>
+            </div>
+            <div class="col-4">
+              <button type="button" class="btn btn-primary mb-2">Select</button>
+            </div>
+          </div>
+        </form>
+        <br>  
+        <form>
+          <div class="form-row">
+            <div class="col-3 date-label">
+              <label>Sunday 7/1</label>
```

Comment:
Are we doing Sundays? If so, our week should be Sun-Saturday. Also check the date. Might be easiest to switch to Saturday 7/4

Reply:
Ok we can do Sun-Saturday and l'll change to Saturday.